labs_api/main.py

```diff
@@ -0,0 +1,52 @@
+import fastapi
+import uvicorn
+from config import Config
+from fastapi.middleware.cors import CORSMiddleware
+from prediction_market_agent_tooling.gtypes import HexAddress
+from prediction_market_agent_tooling.loggers import logger
+
+from labs_api.config import Config
+from labs_api.insights import MarketInsightsResponse, market_insights_cached
+from labs_api.insights_cache import MarketInsightsResponseCache
+
+
+def create_app() -> fastapi.FastAPI:
+    app = fastapi.FastAPI()
+    app.add_middleware(
+        CORSMiddleware,
+        allow_origins=["*"],
+        allow_credentials=True,
+        allow_methods=["*"],
+        allow_headers=["*"],
+    )
```

Comment:
Pretty sure you acknowledged this somewhere - do we have an issue for adding auth? Otherwise our Tavily credits will suffer.

Reply:
It's cached, and the only way to call the endpoint is by using market_id from Omen, anything else will fail before using Tavily. It seemed to me there wasn't a way for someone to abuse this API. Do you see some use-case?

Reply:
Scripting a way to fetch all Omen markets and pinging the endpoint is the only way I can think of.
But yes, it's not very critical.

Reply:
I guess that's the worst-case scenario, where there are many many people on Omen, and they can look at all markets